pixeltable/tests/utils.py

```diff
@@ -221,6 +221,11 @@ def get_video_files(include_bad_video=False) -> List[str]:
         glob_result = [f for f in glob_result if 'bad_video' not in f]
     return glob_result
 
+def get_simple_video_files() -> List[str]:
```

Comment:
What does simple mean?

Are these easier to decode or do they have fewer frames? If the latter, that might be problematic, because it might test fewer corner cases in the code.